app/templates/profile/profile.html

```diff
@@ -24,37 +24,18 @@
             <div class="panel panel-default">
                 <div class="panel-heading">User Profile</div>
                 <div class="panel-body">
-                    {% if new_user %}
-                        <div class="alert alert-warning" role="alert">
-                            Welcome to Hypatio! Please complete your registration form below to get started.
-                        </div>
-                    {% endif %}
-                    {% if not form.email_confirmed.value %}
-                        <div id="email_confirm" class="alert alert-warning" role="alert">
-                            <form id="confirm_email_form" method="post">
-                                {% csrf_token %}
-
-                                Please click the button below to send a verification e-mail. Google's anti-spam measures might ask you to perform a small task before sending the verification e-mail. <br /><br />
-
-                                <script src='https://www.google.com/recaptcha/api.js'></script>
-
-                                <br />
-
-                                <div style="display:none;" id="loadingDiv">
-                                    <img src="/static/default.gif"/>
-                                </div>
-
-                                <button id="submit_verification_email"
-                                        disabled class='btn btn-primary g-recaptcha'
-                                        data-sitekey="{{ recaptcha_client_id }}"
-                                        type="button"
-                                        data-callback="send_confirm_email"
-                                        data-loading-text="Send Verification E-Mail &nbsp;&nbsp; <i class='fa fa-spinner fa-spin '></i>">
-                                    Send Verification E-Mail
-                                </button>
-                            </form>
-                        </div>
-                    {% endif %}
+                {% if new_user %}
+                    <div class="alert alert-warning" role="alert">
+                        Welcome to Hypatio! Please complete your registration form below to get started.
```

Comment:
Nitpicky, can you change this to DBMI portal or something? We'll only call it Hypatio internally. I should have called it something else from the get go.

Reply:
Oh right! Will do.